src/Nethermind/Nethermind.Runner/configs/holesky_archive.cfg

```diff
@@ -1,7 +1,7 @@
 {
     "Init": {
         "ChainSpecPath": "chainspec/holesky.json",
-        "GenesisHash": "0xff9006519a8ce843ac9c28549d24211420b546e12ce2d170c77a8cca7964f23d",
+        "GenesisHash": "0xb5f7f912443c940f21fd611f12828d75b534364ed9e95ca4e307729a4661bde4",
```

Comment:
see this message on holesky channel:


Genesis Event ***
Genesis state root: 0x0ea3f6f9515823b59c863454675fefcd1d8b4f2dbe454db166206a41fda060a0
Genesis block root: 0xab09edd9380f8451c3ff5c809821174a36dce606fea8b5ea35ea936915dbf889
Genesis time: 2023-09-28 12:00:00 GMT

again different hashes?

Reply:
not sure, but this aligns with the config in the holesky repo